nilearn/plotting/displays.py

```diff
@@ -16,6 +16,7 @@
 from matplotlib import lines
 from matplotlib import transforms, colors
```

Comment:
PEP8 + alphabetical order of imports makes it easier to spot import one may be looking for.
```
from matplotlib import (colors,
                       lines,
                       transforms,
                       )

```